admin/css/picker.css

```diff
@@ -34,12 +34,33 @@
 }
 
 .image-picker li[data-wordpress-id]:after {
-  content: "✔️";
-  font-size: 18px;
-  background: white;
+  content: "";
+  display: block;
+  background: url("../../images/added-to-library.svg");
+  width: 25px;
+  height: 25px;
   position: absolute;
-  top: 0;
-  right: 0;
+  /* 2px to account for the border width */
+  top: 2px;
+  left: 2px;
+}
+
+.image-picker .added-to-library {
+  display: none;
+  position: absolute;
```

Comment:
Since by default it's displayed just as `none` I don't think it's needed to compute unnecessary values (I'm referring to lines from 50 to 59).  They should be computed the `wordpress_id` exists and when the user `hover` the picture. What do you think?

Reply:
Sure, makes sense! Updated 👍